reference/build.gradle

```diff
@@ -28,8 +28,21 @@ subprojects {
 		}
 	}
 
+	// :::automatic-testing:3
+	sourceSets {
+		test {
+			compileClasspath += client.compileClasspath
+			runtimeClasspath += client.runtimeClasspath
+		}
+	}
+	// :::automatic-testing:3
+
 	dependencies {
-		modImplementation "net.fabricmc:fabric-loader:0.15.11"
+		modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
+
+		// :::automatic-testing:1
+		testImplementation "net.fabricmc:fabric-loader-junit:${project.loader_version}"
```

Comment:
This should probably be in `latest/build.gradle`? Correct me if im wrong

Reply:
You are right. I'm still a bit confused with this multi-project gradle stuff, thanks for pointing this out.